### PR TITLE
CORE-477 fix the resource-viewer

### PR DESCRIPTION
**Pull request by safydy-r**
The problem:
If we try to enable the resource viewer on community version by adding `resourceViewer` as part of `enabledFeatures` , we got this error.
Screenshot 2023-04-25 142858

It happens because by default we set `visibleBlocks` as null but we call `visibleBlocks.includes(block.uuid)` in a condition

### CODE REVIEW

File: src/components/interaction-designer/ResourceViewer.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <div class="resource-viewer-contents">
       <resource-viewer-block
         v-for="block in activeFlow?.blocks ?? []"
-        v-show="visibleBlocks.includes(block.uuid)"
+        v-show="visibleBlocks === null || visibleBlocks.includes(block.uuid)"
```
Comment (safydy-r) [inline src/components/interaction-designer/ResourceViewer.vue:7]: I'm proposing here to always display those block if the visibleBlocks is null, which would allow us to make a test on that edge case scenario.

**Review by object-required — APPROVED** [review]

**Review by bulatgab — APPROVED** [review]
And we already had the default value for `visibleBlocks` set to `null` 👍
